source/_integrations/adaptive_lighting.markdown

```diff
@@ -0,0 +1,257 @@
+This mechanism works by listening to all `light.turn_on` calls and by noting that the component did not make the call.
+Additionally, there is an option to detect all state changes (when `detect_non_ha_changes` is enabled), so also changes to the lights that were not made by a `light.turn_on` call (e.g., through an app or via something outside of Home Assistant.)
+It does this by comparing a light's state to Adaptive Lighting's previously used settings.
+Whenever a light gets marked as "manually controlled", an `adaptive_lighting.manual_control` event is fired, such that one can use this information in automations.
+
+## Configuration
+
+This integration is both fully configurable through YAML _and_ the frontend. (**Configuration** -> **Integrations** -> **Adaptive Lighting**, **Adaptive Lighting** -> **Options**)
+Here, the options in the frontend and in YAML have the same names.
+
+```yaml
+# Example configuration.yaml entry
+adaptive_lighting:
+  lights:
+    - light.living_room_lights
+```
+
+{% configuration %}
+name:
+  description: The name to use when displaying this switch.
+  required: false
+  default: default
+  type: string
+lights:
+  description: List of light entities for Adaptive Lighting to control (may be empty).
+  required: false
+  type: list
+  default: []
+adapt_brightness:
+  description: Whether Adaptive Lighting should adjust brightness of lights (if supported by the light).
+  required: false
+  default: true
+  type: boolean
+adapt_color_temp:
+  description: Whether Adaptive Lighting should adjust color temperature of lights (if supported by the light).
+  required: false
+  default: true
+  type: boolean
+adapt_rgb_color:
+  description: Whether Adaptive Lighting should adjust RGB color of lights (if supported by the light).
+  required: false
+  default: true
+  type: boolean
+prefer_rgb_color:
+  description: Whether to use RGB color adjustment instead of native light color temperature.
+  required: false
+  default: false
+  type: boolean
+initial_transition:
+  description: How long the first transition is when the lights go from `off` to `on` (or when "sleep mode" is (de)activated).
+  required: false
+  default: 1
+  type: time
+transition:
+  description: How long the transition is when the lights change, in seconds.
+  required: false
+  default: 45
+  type: integer
+interval:
+  description: Period between adapting the lights, in seconds.
+  required: false
+  default: 90
+  type: integer
+min_brightness:
+  description: The minimum percent of brightness to set the lights to.
+  required: false
+  default: 1
+  type: integer
+max_brightness:
+  description: The maximum percent of brightness to set the lights to.
+  required: false
+  default: 100
+  type: integer
+min_color_temp:
```

Comment:
I might be mistaken but I think these are the wrong way around? 2000 is the warmest color, 5500 the coldest? It is a bit confusing because max temperature would suggest warm, but it’s actually the other way around

(See https://medium.com/hd-pro/the-color-temperature-scale-f9184f9d8994)

Reply:
Alternatively, we could consider renaming to `warmest_color_temp` and `coldest_color_temp`, but that breaks the convention of min/ max used for brightness

Reply:
Yeah, I would agree on the numbers part, but would still say this is potentially a bit confusing for the average user. But I’d agree it’s not a main priority to change

In any case, we should make sure to correct the docs by swapping out the words warmest and coldest in the description for min/max, @KTibow :)

Reply:
Ah, I didn't realize that was the case! Thanks for noticing and your feedback!